src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts

```diff
@@ -28,7 +28,8 @@ describe('HmctsGlobalHeaderComponent', () => {
     component = fixture.componentInstance;
     component.headerTitle = {
       name: 'Service name',
-      url: '#'
+      url: '#',
+      hideBranding: true
```

Comment:
It doesn't look like you're testing that this has any effect...?

Reply:
without the test it gives missing property error